changelog/unreleased/issue-14285.toml

```diff
@@ -0,0 +1,5 @@
+type = "c" # One of: a(dded), c(hanged), d(eprecated), r(emoved), f(ixed), s(ecurity)
+message = "Changed event definitions and notifications to new paginated list and entity list ui component."
```

Comment:
Notifications don't use the new component yet - at least I don't see it in the UI, though the backend has the necessary API modifications.

Also, it would be helpful to mention how that changes the behavior, i.e. adds bulk operations. People reading the release notes probably don't care how things are implemented under the hood.